test_helpers_end_to_end/src/snapshot_comparison.rs

```diff
@@ -98,19 +96,7 @@ pub async fn run(
 
     for q in queries.iter() {
         output.push(format!("-- {}: {}", language, q.text()));
-        if q.sorted_compare() {
-            output.push("-- Results After Sorting".into())
-        }
-        if q.normalized_uuids() {
-            output.push("-- Results After Normalizing UUIDs".into())
-        }
-        if q.normalized_metrics() {
-            output.push("-- Results After Normalizing Metrics".into())
-        }
-        if q.normalized_filters() {
-            output.push("-- Results After Normalizing Filters".into())
-        }
-
+        q.add_decription(&mut output);
```

Comment:
I think `add_decription` is supposed to be `add_description`? I'm going through and adding suggestions so you can click them to commit them if that's correct--


```suggestion
        q.add_description(&mut output);
```

Reply:
:facepa